astrosat_users/views/views_customers.py

```diff
@@ -256,7 +280,7 @@ class CustomerUserInviteView(CustomerUserViewMixin, generics.GenericAPIView):
     A special view just for re-sending invitations.
     """
 
-    permission_classes = [IsAuthenticated, IsAdminOrManager]
+    permission_classes = [IsAuthenticated, IsManagerPermission]
     serializer_class = CustomerUserSerializer
 
     def post(self, request, *args, **kwargs):
```

Comment:
Does this allow a manager to re-send an invite to a CustomerUser from any Customer, not just their own Customer(s)?

Reply:
No, it checks that the user making the request is a manager of the customer for whom the request is being made.

Reply:
Okay cheers, that took me a while to understand :)